planner/cascades/integration_test.go

```diff
@@ -33,6 +34,8 @@ type testIntegrationSuite struct {
 }
 
 func newStoreWithBootstrap() (kv.Storage, error) {
+	// TODO: remove this and find out the root cause of race.
```

Comment:
This comment is hard to understand, it's better to add the issue link here

Reply:
done.